public_html/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="es-MX">
+
+<head>
+    <title>Tablero de Reproducibilidad &bull; GECI</title>
+    <meta charset="utf-8">
+
+    <!-- Actualiza tablero porque cambia de forma periódica y automática -->
+    <meta http-equiv="refresh" content="600">
+
+    <!-- Scripts -->
+    <script src="https://code.jquery.com/jquery-3.4.1.min.js"
+        integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
+    <script type="text/javascript" src="js/json2table.js"></script>
+
+    <!-- Estilos -->
+    <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+</head>
+
+<body>
+    <div class="container">
+        <h1>Tablero de la Reproducibilidad</h1>
+        <p>
+            <img src="https://healthchecks.io/badge/3d6fc98b-2ee3-49b9-82ec-45a751/oNv7bkiM/Backup.svg">
+            <img src="https://healthchecks.io/badge/3d6fc98b-2ee3-49b9-82ec-45a751/bYK-iT3x/Inspector.svg">
+            <img src="https://healthchecks.io/badge/3d6fc98b-2ee3-49b9-82ec-45a751/rvBwfTMK/islasgeci.dev.svg">
+            <img src="https://healthchecks.io/badge/3d6fc98b-2ee3-49b9-82ec-45a751/aR22z6k4/Mitotero.svg">
+            <img src="https://healthchecks.io/badge/3d6fc98b-2ee3-49b9-82ec-45a751/sgEiB9kb.svg">
+        </p>
+        <table class="table table-condensed" id="tablero">
+            <tr>
+                <th>Repositorio</th>
+                <th>Objetivo</th>
+                <th>develop</th>
+                <th>master</th>
+            </tr>
+        </table>
+    </div>
+</body>
+
+</html>
```

Comment:
Muy bien hecho!, probado y funcionando de maravilla,
solo me queda este minúsculo detalle de agregar una linea vacía al final de este archivo.
```suggestion
</html>

```